pages/common/xxd.md

```diff
@@ -1,23 +1,23 @@
 # xxd
 
-> Create a hexdump from a binary file or vice-versa.
+> Create a hexadecimal representation (hexdump) from a binary file, or vice-versa.
 
-- Generate a hexdump from a binary file and display the output to console:
+- Generate a hexdump from a binary file and display the output:
 
 `xxd {{input_file}}`
 
-- Send the hexdump output to a file:
+- Generate a hexdump from a binary file and save it to a text file:
 
 `xxd {{input_file}} {{output_file}}`
 
-- Format output in columns of 8 octets per line:
+- Display the output with 10 columns of one octet (byte) each:
 
-`xxd -c {{8}} {{input_file}}`
+`xxd -c {{10}} {{input_file}}`
 
-- Display in plain hex format without any gaps:
+- Display the output in plain mode, without any gaps between the columns:
 
 `xxd -p {{input_file}}`
 
-- Revert a plaintext hex into binary and store it in an output file:
+- Revert a plaintext hexdump into binary, and save it to a binary file:
```

Comment:
"and save it to a file" ? Is the "binary file" necessary since we have already established that it generates binary ?

Reply:
I thought about that while writing this, but I kept it for symmetry with the second example. We can either choose to break the symmetry or make this sentence a little redundant. Which one do you find more reasonable?

Reply:
Ah I see. Sure let's keep the symmetry.